pom.xml

```diff
@@ -131,18 +131,24 @@
         <mockito.version>2.28.2</mockito.version>
         <testng.version>6.14.3</testng.version>
         <log4j.version>1.2.17</log4j.version>
-        <slf4j.version>1.7.26</slf4j.version>
+        <slf4j.version>1.7.28</slf4j.version>
         <logback.version>1.2.3</logback.version>
         <jackson.version>2.9.9</jackson.version>
         <jackson.databind.version>2.9.9.3</jackson.databind.version>
         <kafka.version>2.1.1</kafka.version>
         <janus.version>0.3.1</janus.version>
         <gremlin.version>3.3.3</gremlin.version>
-        <spring-boot.version>2.1.6.RELEASE</spring-boot.version>
+        <spring-boot.version>2.1.9.RELEASE</spring-boot.version>
         <spring-data.version>2.1.9.RELEASE</spring-data.version>
-        <spring-web.version>5.1.8.RELEASE</spring-web.version>
-        <spring-security.version>5.1.5.RELEASE</spring-security.version>
-        <tomcat.version>9.0.21</tomcat.version>
+        <spring-jdbc.version>5.1.8.RELEASE</spring-jdbc.version>
+        <spring-boot-jpa.version>2.1.3.RELEASE</spring-boot-jpa.version>
```

Comment:
The -jpa versions are aligned with spring-data & spring-boot respectively. We need to clarify exactly why we are going out of sync on these versions as there could be other subtle issues (IMO!) by doing so.

Reply:
I agree.
Please check again